flac/register.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 The ZikiChombo Authors. All rights reserved.  Use of this source
+// code is governed by a license that can be found in the License file.
+
+package flac
+
+import (
+	"io"
+
+	"zikichombo.org/codec"
+	"zikichombo.org/sound"
+	"zikichombo.org/sound/sample"
+)
+
+type flacCodec struct {
+	codec.NullCodec
+}
+
+func (f *flacCodec) Extensions() []string {
+	return []string{".flc", ".flac"}
```

Comment:
Didn't know that `.flc` was a recognized extension for FLAC files.

Reply:
I saw that somewhere, don't know where offhand.